### ISSUE TITLE
ValueError: Unable to configure logger 'django_auth_ldap'

### ISSUE THREAD

**Issue by themattinthehatt** [heartexlabs/label-studio#4887]
**Describe the bug**
I cannot launch a fresh installation of label studio due to django logger configuration issues.

**To Reproduce**
1. start a fresh conda environment with python=3.8
2. pip install label-studio
3. in command line, run `label-studio version`. The following error prints out:

```
=> Database and media directory: /teamspace/studios/this_studio/.local/share/label-studio
=> Static URL is set to: /static/
=> Database and media directory: /teamspace/studios/this_studio/.local/share/label-studio
=> Static URL is set to: /static/
/teamspace/studios/this_studio/.local/share/label-studio/.env not found - if you're not configuring your environment separately, check this.
get 'SECRET_KEY' casted as '<class 'str'>' with default ''
Warning: SECRET_KEY not found in environment variables. Will generate a random key.
[Tracing] Create new propagation context: {'trace_id': '5eb45667408340cab988e2aa8c9d69b0', 'span_id': 'b4f7a073bbd8f55d', 'parent_span_id': None, 'dynamic_sampling_context': None}
Starting new HTTPS connection (1): pypi.org:443
https://pypi.org:443 "GET /pypi/label-studio/json HTTP/1.1" 200 59014
Traceback (most recent call last):
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.8/logging/config.py", line 617, in configure
    self.configure_logger(name, loggers[name])
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.8/logging/config.py", line 794, in configure_logger
    self.common_logger_config(logger, config, incremental)
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.8/logging/config.py", line 779, in common_logger_config
    logger.setLevel(logging._checkLevel(level))
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.8/logging/__init__.py", line 198, in _checkLevel
    raise ValueError("Unknown level: %r" % level)
ValueError: Unknown level: 'info'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/zeus/miniconda3/envs/cloudspace/bin/label-studio", line 8, in <module>
    sys.exit(main())
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.8/site-packages/label_studio/server.py", line 301, in main
    _setup_env()
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.8/site-packages/label_studio/server.py", line 40, in _setup_env
    get_wsgi_application()
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.8/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.8/site-packages/django/utils/log.py", line 75, in configure_logging
    logging_config_func(logging_settings)
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.8/logging/config.py", line 808, in dictConfig
    dictConfigClass(config).configure()
  File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.8/logging/config.py", line 619, in configure
    raise ValueError('Unable to configure logger '
ValueError: Unable to configure logger 'django_auth_ldap'
Starting new HTTPS connection (1): o227124.ingest.sentry.io:443
https://o227124.ingest.sentry.io:443 "POST /api/5820521/envelope/ HTTP/1.1" 200 41
```

**Expected behavior**
Print the version of label studio and then terminate.

**Environment**
 - OS: Ubuntu 20.04.6
 - Label Studio Version: 1.9.1

**Comment by jombooth** [comment]
Hi @themattinthehatt,

I tried to reproduce this with a fresh Conda env using python 3.8 but wasn't able to; `label-studio version` worked fine for me with 1.9.1. However, if I ran `LOG_LEVEL=info label-studio version` I got the same crash as you did.

Label Studio will configure the log level for `django_auth_ldap` based on the environment variable `LOG_LEVEL`, and it defaults to `DEBUG`:

https://github.com/HumanSignal/label-studio/blob/adc2475e6089c57ff23eb4a933890e43762176e7/label_studio/core/settings/base.py#L57

is it possible you have `LOG_LEVEL` set to `info` (lowercase) for some non-Label-Studio-related reason? In any case, running the command like so should work:

```
LOG_LEVEL=DEBUG label-studio version
```
